Doc/library/imghdr.rst

```diff
@@ -16,8 +16,8 @@ The :mod:`imghdr` module defines the following function:
 
 .. function:: what(file, h=None)
 
-   Tests the image data contained in the file named by *file*, and returns a
-   string describing the image type.  If optional *h* is provided, the *file*
+   Tests the image data contained in the file named *file*, and returns a
```

Comment:
Perhaps use the imperative form of the verbs, that is, `Test` and `return`?  Also, the comma before the `and` isn't needed.

Reply:
I find Docs use *Tests* and Doc strings uses *Test*